### PR TITLE
PLAT-128528: Migrate hello enact and kitten browser tutorials to sandstone

### PR DESCRIPTION
**Pull request by YunhoLee0**
Enact-DCO-1.0-Signed-off-by: Yunho Lee (yunho2.lee@lge.com)

Merge feature/kittenbrowser-migration branch to develop branch

### CODE REVIEW

**Review by MikyungKim — APPROVED** [review]
LGTM

File: tutorial-kitten-browser/src/components/Kitten/Kitten.module.less [hunk 1]
```diff
@@ -1,5 +1,12 @@
+@import "~@enact/sandstone/styles/mixins.less";
+
 .kitten {
 	display: inline-block;
-	padding: 12px;
+	padding: 22px;
```
Comment (MikyungKim) [inline tutorial-kitten-browser/src/components/Kitten/Kitten.module.less:5]:
```suggestion
	padding: 24px;
```

Comment (YunhoLee0) [reply tutorial-kitten-browser/src/components/Kitten/Kitten.module.less]: 🙆‍♂️

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**